staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -110,7 +110,7 @@ func NewCmdScale(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobr
 	cmd := &cobra.Command{
 		Use:                   "scale [--resource-version=version] [--current-replicas=count] --replicas=COUNT (-f FILENAME | TYPE NAME)",
 		DisableFlagsInUseLine: true,
-		Short:                 i18n.T("Set a new size for a Deployment, ReplicaSet or Replication Controller"),
+		Short:                 i18n.T("Set a new size for a Deployment, ReplicaSet, Replication Controller or StatefulSet."),
```

Comment:
/lgtm cancel
If you want to fix it, let's fix the entire description of this command such that it talks about the possibility to scale any resource having `/scale` sub-resource. As an example calling out the above.

Reply:
No more sub-resources to scale?
`Short` description is consistent with `Long` description
https://github.com/kubernetes/kubernetes/blob/66cbf0196bd5ed0f06d7c40fccba887aae5405d8/staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go#L39-L41